src/Uno.Toolkit.UI/Behaviors/StatusBar.iOS.cs

```diff
@@ -21,9 +21,13 @@ public static partial class StatusBar
 			// random unique tag to avoid recreating the view
 			const int StatusBarViewTag = 38482;
 
+#pragma warning disable CA1416 // This call site is reachable on: 'iOS' 15.4 and later, 'maccatalyst' 15.4 and later. 'UIApplication.Windows.get' is unsupported on: 'ios' 15.0.0 and later, 'maccatalyst' 15.0.0 and later.
```

Comment:
This generally means that there should be an explicit validation of this piece of code against iOS 14 and earlier.

Reply:
The APIs documentation mention replacements to use. I attempted to use them in the last commit.
But if there is not enough testing in this repo, it's best that someone test the changes.

Reply:
@kazo0 will you able to take a look?